### PR TITLE
[PUB-2993] Replace features.isFreeUser() by org features

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
Replace features.isFreeUser() and features.isProUser() by orgs features:
- `hasApprovalFeature: orgData.planBase === 'business',`
- `hasDraftsFeature: orgData.planBase !== 'free',`
-  `hasGridFeature: orgData.planBase === 'business',`
-  `hasStoriesFeature: orgData.planBase === 'business',`

## Context & Notes
https://buffer.atlassian.net/browse/PUB-2993
In the future I see tabs as something that doesn't have this weird complex logic of grabbing utils functions. For each profile we would get an array of pages that would be used across the app. But it gets out of the scope of this task.

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/server/parsers/src/orgParser.js [hunk 1]
```diff
@@ -27,5 +27,11 @@ module.exports = orgData => ({
   hasShareAgainFeature: orgData.planBase !== 'free',
   hasCustomizingUtmParamsFeature:
     orgData.planBase === 'business' && orgData.isOwner,
+  hasApprovalFeature: orgData.planBase === 'business',
```
Comment (albennett) [inline packages/server/parsers/src/orgParser.js:30]: I can foresee this feature list growing long over time. Curious on your thoughts around moving the `hasFeature` items into an array for the org store.

**Review by albennett — APPROVED** [review]
Great job @anafilipadealmeida 🥇

File: packages/server/parsers/src/orgParser.js (hunk 1 above)
Comment (anafilipadealmeida) [reply packages/server/parsers/src/orgParser.js:30]: In the future I definitely see this coming from an array of features, either grouped here or already coming as an array from the API! The only thing still not clear in my mind is what to call it, as right now we kind of mix feature flips, plan features and role features 😅

**Pull request merged**